loom/script/runtime/lsLuaState.cpp

```diff
@@ -86,8 +85,8 @@ void LSLuaState::open()
 {
     assert(!L);
 
-    L = lua_newstate(lsLuaAlloc, NULL);
-    //L = luaL_newstate();
+    //L = lua_newstate(lsLuaAlloc, NULL);
+    L = luaL_newstate();
```

Comment:
This is the default luajit codepath: https://github.com/LoomSDK/LoomSDK/blob/dfe41a7b508b23c4695a40023ebd523b1c4ea655/loom/vendor/luajit/src/lj_alloc.c#L1369 Assuming we don't define "use system allocator" Seems clever?